fmn/api/main.py

```diff
@@ -51,6 +56,17 @@ async def read_root(
     return result
 
 
+@app.get("/users/")
+def get_users():  # pragma: no cover
```

Comment:
Look at the `read_root` function for arguments to require the user's credentials and identity.

Reply:
read root wasnt working for me anymore on my tinystage instance -- i tired to see what was causing the issue but to no avail